.github/workflows/main.yaml

```
@@ -37,18 +37,12 @@ jobs:
     - uses: actions/download-artifact@v3
       with:
         name: src
-    - uses: AbsaOSS/k3d-action@v2
-      with:
-        cluster-name: "kanister-run-${{ matrix.testSuite }}"
-        args: >-
-          --agents 3
-          --no-lb
-          --k3s-arg "--no-deploy=traefik,servicelb@server:*"
+    - uses: helm/kind-action@v1.4.0
     - run: tar -xvf ./src.tar.gz
     - run: |
         make install-csi-hostpath-driver
         make install-minio
-      if: matrix.testSuite == 'integration-test' || matrix.testSuite == 'helm-test'
+      if: matrix.testSuite == 'integration-test' || matrix.testSuite == 'helm-test' || matrix.testSuite == 'test'
```

I believe this change is not required - we don't need minio and hostpath driver for unit tests.
```suggestion
      if: matrix.testSuite == 'integration-test' || matrix.testSuite == 'helm-test'
```